API/Controllers/WizardController.cs

```diff
@@ -57,9 +53,8 @@ public WizardController(IMapper mapper, SourceManagerService sourceManagerServic
         /// <returns></returns>
```

Comment:
Empty returns tag (while this method does return something).
Please, also fix the other empty returns tags in this file :)